src/main/scala/nl/knaw/dans/easy/fsrdb/FsRdbUpdater.scala

```diff
@@ -68,7 +69,9 @@ object FsRdbUpdater {
         mimetype = (metadata \ "mimeType").text,
         creatorRole = (metadata \ "creatorRole").text,
         visibleTo = (metadata \ "visibleTo").text,
-        accessibleTo = (metadata \ "accessibleTo").text)
+        accessibleTo = (metadata \ "accessibleTo").text,
+        sha1Checksum = if((fileDS \ "datastreamVersion" \ "contentDigest" \ "@DIGEST").size > 0) (fileDS \ "datastreamVersion" \ "contentDigest" \ "@DIGEST").text
```

Comment:
This is ugly, but I can't see a better way to do it. If I try to get the digest attribute inside the for-comprehension and it doesn't exist (a legitimate case) then the whole for-comprehension will return zero results.

@GeorgiKhomeriki : what says the flatMap-expert?